tests/koji/test_koji_vmi.py

```diff
@@ -10,6 +10,7 @@
     AmiRelease,
     VMIRelease,
 )
+from pushsource._impl.model.base import KojiBuildInfo
```

Comment:
Can you please import it from `pushsource` above like all the rest of the imports?

Reply:
Done